README.md

```diff
@@ -79,21 +79,18 @@ Since mu_environment has a dependency on mu_python_library, it should be updated
 
 # Setup Info for iMX8
 
-- `sudo apt-get install gcc g++ make python3 python3-pip git`
+- `sudo apt-get install gcc g++ make python3 python3-pip git mono-devel`
     - Minimum Python version is 3.6
     - Minimum git version 2.11
 - Install pip modules for build:
-    - `pip3 install mu_environment`
+    - `pip3 install -r requirments.txt`
 - `git clone https://windowspartners.visualstudio.com/MSCoreUEFI/MSCoreUEFI%20Team/_git/NXP_iMX_Platform`
```

Comment:
https://github.com/ms-iot/MU_PLATFORM_NXP is our official branch.  This should be updated.